.teamcity/settings.kts

```diff
@@ -37,275 +37,284 @@ version = "2020.1"
+		password("matticbot_oauth_token", "credentialsJSON:34cb38a5-9124-41c4-8497-74ed6289d751", display = ParameterDisplay.HIDDEN)
+		param("teamcity.git.fetchAllHeads", "true")
+		text("env.CHILD_CONCURRENCY", "15", label = "Yarn child concurrency", description = "How many packages yarn builds in parallel", allowEmpty = true)
+		text("docker_image", "registry.a8c.com/calypso/ci:latest", label = "Docker image", description = "Docker image to use for the run", allowEmpty = true)
+	}
+
+	features {
+		dockerRegistry {
+			id = "PROJECT_EXT_6"
+			name = "Docker Registry"
+			url = "https://registry.a8c.com"
+		}
+		githubConnection {
+			id = "PROJECT_EXT_8"
+			displayName = "GitHub.com"
+			clientId = "abfe9b6b38deb65e68e5"
+			clientSecret = "credentialsJSON:52797023-03f8-430a-b66f-2ac50fcc9608"
+		}
+	}
 }
 
 object BuildBaseImages : BuildType({
-    name = "Build base images"
-    description = "Build base docker images"
-
-    buildNumberPattern = "%build.prefix%.%build.counter%"
-
-    params {
-        param("build.prefix", "1.0")
-    }
-
-    vcs {
-        root(WpCalypso)
-
-        cleanCheckout = true
-        branchFilter = """
-            +:*
-            +:wip/docker-for-ci
-        """.trimIndent()
-        excludeDefaultBranchChanges = true
-    }
-
-    steps {
-        script {
-            name = "Build docker images"
-            scriptContent = """
-                VERSION="%build.number%"
-                BUILDER_IMAGE_NAME="registry.a8c.com/calypso/base"
-                CI_IMAGE_NAME="registry.a8c.com/calypso/ci"
-                BUILDER_IMAGE="${'$'}{BUILDER_IMAGE_NAME}:${'$'}{VERSION}"
-                CI_IMAGE="${'$'}{CI_IMAGE_NAME}:${'$'}{VERSION}"
-                
-                docker build -f docker/Dockerfile --no-cache --target builder -t "${'$'}BUILDER_IMAGE" .
-                docker build -f docker/Dockerfile --target ci -t "${'$'}CI_IMAGE" .
-                
-                docker tag "${'$'}BUILDER_IMAGE" "${'$'}{BUILDER_IMAGE_NAME}:latest"
-                docker tag "${'$'}CI_IMAGE" "${'$'}{CI_IMAGE_NAME}:latest"
-                
-                docker push "${'$'}CI_IMAGE"
-                docker push "${'$'}{CI_IMAGE_NAME}:latest"
-                
-                docker push "${'$'}BUILDER_IMAGE"
-                docker push "${'$'}{BUILDER_IMAGE_NAME}:latest"
-            """.trimIndent()
-            dockerImagePlatform = ScriptBuildStep.ImagePlatform.Linux
-            dockerRunParameters = "-u %env.UID%"
-        }
-    }
-
-    features {
-        perfmon {
-        }
-        dockerSupport {
-            loginToRegistry = on {
-                dockerRegistryId = "PROJECT_EXT_6"
-            }
-        }
-    }
+	name = "Build base images"
+	description = "Build base docker images"
+
+	buildNumberPattern = "%build.prefix%.%build.counter%"
+
+	params {
+		param("build.prefix", "1.0")
+	}
+
+	vcs {
+		root(WpCalypso)
+
+		cleanCheckout = true
+		branchFilter = """
+			+:master
```

Comment:
This means that this build ("Build base images") will only build from master. The idea is that we don't want random branches pushing images to our registry tagged as `:latest`

We may change this in the future to have branches generate an image tagged as `:<branch-name>-<version>` or something like that.